### ISSUE TITLE
Batchsize of QAT

### ISSUE THREAD

**Issue by sf-Liu** [quic/aimet#3267]
Hi, I'm trying the QAT. 

When I do the training, I have to make the batch size more than 1, or the BatchNorm will report errors.
But my target is to generate a quantized model with only 1 input batch, is there any way to change the input batch size of `sim.model`? Or any method to satisfy my target?

**Comment by quic-klhsieh** [comment]
@sf-Liu , it's possible to perform QAT training in quantsim with batch size > 1 but then use a dummy_input with batch_size = 1 when exporting the quantsim to target. Can you try this out and reopen this ticket if more assistance is needed?
